handlers/handlers.go

```diff
@@ -90,6 +91,23 @@ func handleDefault(r *ranch.Ranch) http.HandlerFunc {
 	}
 }
 
+var (
+	acquireWaitingDurationSeconds = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "boskos_acquire_waiting_duration_seconds",
```

Comment:
nit: I think you can remove `waiting` from this - `boskos_acquire_duration_seconds` seems sufficient to me.

alternately could make this `boskos_request_duration_seconds` if we think we might want to instrument other handlers later, with an additional `verb` label.

Reply:
Don't we already have the latter? For actual HTTP request durations

Reply:
the existing HTTP metrics don't include query parameters (like type, state, dest), which is I presume the intent of this PR.